app/src/components/Order.tsx

```diff
@@ -36,14 +86,32 @@ class Order extends Component<Props> {
                 classes={{ tooltip: classes.offsetTooltip }}
             >
                 <Box
-                    className={orderClasses}
+                    className={orderInfoDrawerShown
+                    && orderIdFromState === orderId ? clickedOrderClass : orderClasses}
                     style={{ width: `${quantityBoxSize}%` }}
+                    onClick={this.handleOnOrderClick}
                 >
-                    {shouldShowQuantity && <Typography className={classes.text}>{quantity}</Typography>}
+                    {shouldShowQuantity && (
+                        <Typography className={orderInfoDrawerShown
+                    && orderIdFromState === orderId ? classes.orderTextOnClick : classes.text}
+                        >
+                            {quantity}
+                        </Typography>
+                    )}
                 </Box>
             </Tooltip>
         );
     }
 }
 
-export default withStyles(styles)(Order);
+const mapStateToProps = (state: RootState) => ({
+    orderInfoDrawerShown: state.general.showOrderInfoDrawer,
+    orderIdFromState: state.general.orderDetails ? state.general.orderDetails.id : -1,
+});
+
+const mapDispatchToProps = (dispatch : Dispatch) => ({
+    // eslint-disable-next-line max-len
```

Comment:
just break the line instead of suppressing it

Reply:
done